src/get-addon-metadata.js

```diff
@@ -24,22 +24,22 @@ function promisedRequest(url) {
 
 module.exports = async function(addonNames) {
   let requests = addonNames.map((addon) => {
-    return promisedRequest(`https://emberobserver.com/api/v2/addons?filter%5Bname%5D=${addon}&page%5Blimit%5D=1`)
+    addon = addon.replace(/\@|\/|\./g, '-')
+    return promisedRequest(`https://emberobserver.com/badges/${addon}.svg`);
   });
 
   let responses = await Promise.all(requests);
-
-  let successfulResponses = responses.filter((response) => {
-    return !response.errors;
-  })
+  let successfulResponses = responses;
```

Comment:
Unnecessary assignment

Reply:
Thanks! Reminds me I need to get linting set up.

Reply:
Thanks! Reminds me I need to get linting set up.